File: python/pole_prostokat.py
#!/usr/bin/env python
# -*- coding: utf-8 -*-
#
#  pole_prostokat.py



def main(args):
    bok1 = int(input("Podaj pierwszy bok: "))
    bok2 = int(input("Podaj drugi bok: "))
    pole = bok1 * bok2
    print("Pole prostokąta wynosi", pole)
    obw = bok1 + bok2 + bok1 + bok2
    print("Obwód prostokąta wynosi", obw)
    
    return 0

if __name__ == '__main__':
    import sys
    sys.exit(main(sys.argv))

File: cpp/pole_prostokat.cpp
/*
 * pole_kwadrat.cpp
 */


#include <iostream>

using namespace std;

int main(int argc, char **argv)
{
	int bok1;  
	int bok2;       
	int pole;
	int obw;
	cout << "Podaj pierwszy bok prostokąta: ";
	cin >> bok1;
	cout << "Podaj drugi bok prostokąta: ";
	cin >> bok2;
	pole = bok1 * bok2;
	obw = bok1+bok1+bok2+bok2;
	cout << "Pole prostokąta wynosi: " << pole << endl;
	cout << "Obwód prostokąta wynosi: " << obw;
	
	
	return 0;
}
